test/e2e/const.go

```diff
@@ -110,7 +119,8 @@ const (
 
 	AuthSecretName = "basic-auth-secret"
 
-	ShortTestLabel = "short"
-	FullTestLabel  = "full"
-	DontRunLabel   = "dontrun"
+	ShortTestLabel        = "short"
+	FullTestLabel         = "full"
+	DontRunLabel          = "dontrun"
+	LocalvSphereTestLabel = "local"
```

Comment:
```suggestion
	LocalTestLabel = "local"
```

There may be other scenarios where tests can only be run locally...not just vpshere.